src/NexusMods.App.UI/Pages/LoadoutPage/LoadoutViewModel.cs

```diff
@@ -135,24 +137,28 @@ public LoadoutViewModel(
 
             RulesSectionViewModel = new SortingSelectionViewModel(serviceProvider, windowManager, loadoutId, canEditObservable: isSingleCollectionObservable);
 
-            CommandUploadRevision = new ReactiveCommand<Unit>(async (unit, cancellationToken) =>
+            CommandShareCollection = IsCollectionUploaded.Select(static b => !b).ToReactiveCommand<Unit>(async (_, cancellationToken) =>
```

Comment:
When should these commands be enabled? Only if you already have the collection uploaded, but that seems wrong?

Reply:
Missed the negation in the select, was confused because of that.